barretenberg/cpp/scripts/benchmark_wasm_remote_wasmer.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
```

Comment:
I added this to run a different wasm runtime as an alternative source of truth for the speedups